include/universal/utility/error.hpp

```diff
@@ -35,5 +36,41 @@ ReturnType RelativeError(const ArgumentType& actual, const ArgumentType& referen
 	return (ReturnType(actual) - ReturnType(reference)) / ReturnType(reference);
 }
 
-}} // namespace sw::universal
+/// <summary>
+/// The logarithmic relative error is defined as the difference of the computed value to the reference value.
+/// Using this method we can determine error on a logarithmic scale.
+/// The logarithmic relative error gives an indication of how good the answer is relative to the value being approximated on a log scale.
+/// </summary>
+/// <typeparam name="ArgumentType">type representation of the value and reference</typeparam>
+/// <typeparam name="ReturnType">type representation of relative error. defaults to double</typeparam>
+/// <param name="actual">measured or computed value</param>
+/// <param name="reference">reference value of the quantity</param>
+/// <returns>Log error between actual and reference values</returns>
+template<typename ArgumentType, typename ReturnType = double>
+ReturnType LogRelativeError(const ArgumentType& actual, const ArgumentType& reference){ 
+    ReturnType a = std::abs(ReturnType (actual));
+    ReturnType r = std::abs(ReturnType (reference)); 
+    using std::log10;
+    return ReturnType (log10(a) - log10(r));
+}
+
+
+/// <summary>
+///
+/// </summary>
+/// <typeparam name="ArgumentType">type representation of the value and reference</typeparam>
+/// <typeparam name="ReturnType">type representation of relative error. defaults to double</typeparam>
+
+/// <returns>relative error between actual and reference values</returns>
+template<typename ReturnType = double>
```

Comment:
the template parameter ArgumentType has been removed erroneously. The design of this template function makes the argument types and the return type programmable to the context. Please add ArgumentType back into this function definition.